### ISSUE TITLE
[Bug] setVisibleThroughObjects does not work with addColor

### ISSUE THREAD

**Issue by hohserg1** [StarChasers/OCGlasses#100]
**Describe the bug**
Using addColor neutralizes effect of setVisibleThroughObjects(true)

**In which environment did the Bug appear?**
Singleplayer

**To Reproduce**
1. Open lua repl
2. execute `c=component.glasses.addcube3D()`
3. execute `c.setVisibleThroughObjects(true)
4. See cube through blocks
5. execute `c.addColor(1,0,0,0.5) --red, half transparent`
6. Do not see cube through blocks

**Expected behavior**
Cube must visible through blocks independent on color

**Screenshots / Code Snippet**
After step 3:
![image](https://user-images.githubusercontent.com/35610168/104472143-b303eb80-55cc-11eb-9338-c5215838463a.png)
After step 5:
![image](https://user-images.githubusercontent.com/35610168/104472286-d890f500-55cc-11eb-9169-6db95f4d467a.png)

**Minecraft:**
 - Mod Version - OpenGlasses-MC1.12.2-2.2-53
 - Minecraft Version - 1.12.2
 - Forge Version - 14.23.5.2854

**Comment by hohserg1** [comment]
I tested it better:
it bug work with any widgets and work only if alpha modifier less that 1

**Comment by ben-mkiv** [comment]
duplicate of https://github.com/Starchasers/OCGlasses/issues/91
